src/temp/ProjectInfo/ContactDetails.tsx

```diff
@@ -0,0 +1,94 @@
+import LocationIcon from '../icons/LocationIcon';
+import MailIcon from '../icons/MailIcon';
+import RightArrowIcon from '../icons/RightArrowIcon';
+import { ViewProfileIcon } from '../icons/ViewProfileIcon';
+import WebsiteLinkIcon from '../icons/WebsiteLinkIcon';
+import styles from './ProjectInfo.module.scss';
+import { useTranslation } from 'next-i18next';
+import SingleProjectInfoItem from './SingleProjectInfoItem';
+
+interface Props {
+  websiteURL: string;
+  location: string;
+  email: string;
+}
+
+const ContactDetails = ({ websiteURL, location, email }: Props) => {
+  const { t } = useTranslation(['donate']);
+
+  const extractWebsiteURLTitle = () => {
+    return websiteURL
+      .replace('https://www.', '')
+      .replace('http://', '')
+      .replace('https://', '')
+      .replace('https://www.', '')
+      .split(/[/?#]/)[0];
+  };
+
+  const contactDetails = [
+    {
+      icon: (
+        <ViewProfileIcon
+          width={10.5}
+          color={`${'var(--primary-font-color)'}`}
+        />
+      ),
+      title: t('donate:viewProfile'),
+      link: '',
+    },
+    {
+      icon: (
+        <WebsiteLinkIcon
+          width={10.5}
+          color={`${'var(--primary-font-color)'}`}
+        />
+      ),
+      title: extractWebsiteURLTitle(),
+      link: websiteURL,
+    },
+    {
+      icon: (
+        <LocationIcon width={9.5} color={`${'var(--primary-font-color)'}`} />
+      ),
+      title: location,
+      link: '',
+    },
+    {
+      icon: <MailIcon width={10.5} color={`${'var(--primary-font-color)'}`} />,
+      title: email,
+      link: `mailto:${email}`,
+    },
+  ];
+
+  return (
+    <div className={styles.contactDetailsContainer}>
+      <SingleProjectInfoItem
+        title={t('donate:contactDetails')}
+        itemContent={
```

Comment:
Perhaps not as a part of this PR, but we should keep the translations for this page within one file.

Reply:
I have added new files `allprojects.json` and `projectDetail.json`, I could all the translations in the second file. Do I need to do that for this component specifically or all the components under projectinfo folder

Reply:
I think this can be done at one go off the base PR.

Reply:
I've added a task for this in JIRA.